components/gif_picker/components/InfiniteScroll/index.ts

```diff
@@ -47,28 +49,22 @@ export default class InfiniteScroll extends PureComponent {
             children,
             element,
             hasMore,
-            initialLoad, // eslint-disable-line no-unused-vars
             loader,
-            loadMore, // eslint-disable-line no-unused-vars
-            pageStart, // eslint-disable-line no-unused-vars
-            threshold, // eslint-disable-line no-unused-vars
-            useWindow, // eslint-disable-line no-unused-vars
-            isReverse, // eslint-disable-line no-unused-vars
-            scrollPosition, // eslint-disable-line no-unused-vars
             containerHeight,
             ...props
         } = this.props;
 
-        props.ref = (node) => {
+        props.ref = (node: React.ReactNode) => {
             this.scrollComponent = node;
         };
 
         const elementProps = containerHeight ? {...props, style: {height: containerHeight}} : props;
 
-        return React.createElement(element, elementProps, children, hasMore && (loader || this.defaultLoader));
+        return React.createElement(element!, elementProps, children, hasMore && (loader || this.defaultLoader));
     }
 
-    calculateTopPosition(el) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    calculateTopPosition = (el: { offsetTop?: any; offsetParent?: any }): any => {
```

Comment:
Since there is a check for el not existing, el should be marked as optional ie `el?:`